src/containers/Devices/components/DevicesList.js

```diff
@@ -159,38 +157,47 @@ export default class DevicesList extends Component {
 
                 await this.props.glpi.deleteItem({ itemtype: 'PluginFlyvemdmAgent', input: itemListToDelete, queryString: { force_purge: true } })
 
-                this.props.showNotification('Success', 'elements successfully removed')
-                this.props.changeAction(null)
-                this.props.changeSelectionMode(false)
-                this.setState({
-                    selectedItemList: []
+                this.props.setNotification({
+                    title: 'Successfully',
+                    body: 'Device successfully removed!',
+                    type: 'success'
                 })
+
+                this.setState((prevState, props) => ({
+                    selectedItems: [],
+                    selectionMode: !prevState.selectionMode,
+                    isLoading: false
```

Comment:
Why you dont dispatch the action for change loading state? uiTransactionStart and uiTransactionFinish

Reply:
Yes, I'll use it